### ISSUE TITLE
Heading has accessible name - ffd0e9 - Failed example 1

### ISSUE THREAD

**Issue by joao-vicente** [act-rules/act-rules.github.io#1110]
The Failed Example 1 in rule [https://act-rules.github.io/rules/ffd0e9m](https://act-rules.github.io/rules/ffd0e9m) is wrong. As you can see in [https://www.w3.org/TR/html-aam-1.0/#section-elements-and-grouping-content-elements-not-listed-elsewhere](https://www.w3.org/TR/html-aam-1.0/#section-elements-and-grouping-content-elements-not-listed-elsewhere), if the element has an `aria-label` then you must use the algorithm defined in [https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te) 2C, which states that if the `arial-label` is empty (""), then it should be ignored. So by following the algorithm the accessible name is derived by the text from the node. This example is similar to the Passed Example 1.

**Comment by Jym77** [comment]
You are right.

**Comment by Jym77** [comment]
Closed by #1116.
